src/jdk.incubator.vector/share/classes/jdk/incubator/vector/VectorOperators.java

```diff
@@ -563,9 +563,36 @@ static boolean opKind(Operator op, int bit) {
     public static final /*bitwise*/ Associative OR = assoc("OR", "|", VectorSupport.VECTOR_OP_OR, VO_NOFP+VO_ASSOC);
     /*package-private*/ /** Version of OR which works on float and double too. */
     static final Associative OR_UNCHECKED = assoc("OR_UNCHECKED", "|", VectorSupport.VECTOR_OP_OR, VO_ASSOC+VO_PRIVATE);
+
     /** Produce {@code a^b}.  Integral only. */
     public static final /*bitwise*/ Associative XOR = assoc("XOR", "^", VectorSupport.VECTOR_OP_XOR, VO_NOFP+VO_ASSOC);
 
+    /** Produce saturating {@code a+b}.  Integral only.
+     * @see VectorMath#addSaturating(int, int)
+     */
+    public static final Associative SADD = assoc("SADD", "+", VectorSupport.VECTOR_OP_SADD, VO_NOFP+VO_ASSOC);
+    /** Produce saturating unsigned {@code a+b}.  Integral only.
+     * @see VectorMath#addSaturatingUnsigned(int, int)
+     */
+    public static final Associative SUADD = assoc("SUADD", "+", VectorSupport.VECTOR_OP_SUADD, VO_NOFP+VO_ASSOC);
+    /** Produce saturating {@code a-b}.  Integral only.
+     * @see VectorMath#subSaturating(int, int)
+     */
+    public static final Binary SSUB = binary("SSUB", "-", VectorSupport.VECTOR_OP_SSUB, VO_NOFP);
+    /** Produce saturating unsigned {@code a-b}.  Integral only.
+     * @see VectorMath#subSaturatingUnsigned(int, int)
+     */
+    public static final Binary SUSUB = binary("SUSUB", "-", VectorSupport.VECTOR_OP_SUSUB, VO_NOFP);
+    /** Produce unsigned {@code min(a,b)}.  Integral only.
+     * @see VectorMath#minUnsigned(int, int) (int, int)
+     */
+    public static final Associative UMIN = assoc("UMIN", "umin", VectorSupport.VECTOR_OP_UMIN, VO_NOFP+VO_ASSOC);
```

Comment:
We should rename the existing unsigned compare operators to use the same naming scheme i.e., s/UNSIGNED_LT/ULT etc.